### PR TITLE
Adds lethal injection syringe to research and seclathe

### PR DESCRIPTION
**Pull request by KoboldCommando**
# About The Pull Request

This allows you to print more lethal injection syringes at (only) the security lathe, after you've researched the "medical weaponry" node.  The price is based on a metamaterial beaker and bluespace syringe.

## Why It's Good For The Game

I'm not a fan of how the lethal syringe is a unique item.  I think it should either be able to stand on its own, or be brought in line balance-wise.  Possible moves forward if it feels too strong could include giving it a longer injection time, and making it impossible to load into various syringe launchers.  The requirement of research, plastic, gold and titanium should limit mass-production.

## Testing Photographs and Procedure
Checked lathes, no syringe.  Did research.  Checked all lathes again, no syringe in the science or medbay lathe, rapid syringe gun was in the medbay lathe, lethal syringe was in the seclathe and rapid syringe gun was not.

## Changelog

:cl:
add: Lethal injection syringes can now be researched, and are printed at the security lathe
del: Syringe guns can no longer load lethal injection or bluespace syringes
/:cl:

### CODE REVIEW

**Comment by nednaZ** [comment]
I'm only in favor of this if it cannot be loaded into a syringe gun.

**Comment by KoboldCommando** [comment]
> I'm only in favor of this if it cannot be loaded into a syringe gun. Personally I'd like the bluespace syringe to be unloadable as well

Done!  I was already considering going ahead and doing both already.  So it's done.  No more big syringes in syringe guns!  Normal/crude/cryo/piercing/etc are still fine.

File: code/modules/projectiles/guns/misc/syringe_gun.dm [hunk 1]
```diff
@@ -61,6 +61,9 @@
 
 /obj/item/gun/syringe/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
 	if(istype(A, /obj/item/reagent_containers/syringe))
+		if(istype(A, /obj/item/reagent_containers/syringe/lethal) || istype(A, /obj/item/reagent_containers/syringe/bluespace)) //monkestation edit: no more lethal or bluespace syringes
```
Comment (Zonespace27) [inline code/modules/projectiles/guns/misc/syringe_gun.dm:64]: making it a var on the syringe is probably better

File: code/modules/research/designs/medical_designs.dm [hunk 2]
```diff
@@ -62,6 +62,18 @@
 	build_path = /obj/item/reagent_containers/glass/beaker/meta
 	category = list("Medical Designs")
 
+//monkestation edit begin
```
Comment (Zonespace27) [inline code/modules/research/designs/medical_designs.dm:65]: gonna want to put this in modular_monkestation

File: code/modules/projectiles/guns/misc/syringe_gun.dm (hunk 1 above)
Comment (KoboldCommando) [reply code/modules/projectiles/guns/misc/syringe_gun.dm]: Good suggestions!  Both implemented.

File: code/modules/projectiles/guns/misc/syringe_gun.dm [hunk 3]
```diff
@@ -61,6 +61,10 @@
 
 /obj/item/gun/syringe/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
 	if(istype(A, /obj/item/reagent_containers/syringe))
+		var/obj/item/reagent_containers/syringe/S = A //to check shootability //monkestation edit
```
Comment (nednaZ) [inline code/modules/projectiles/guns/misc/syringe_gun.dm:64]: More descriptive variable names would be appreciated

File: monkestation/code/modules/research/designs/medical_designs.dm [hunk 4]
```diff
@@ -19,3 +19,13 @@
 	build_path = /obj/item/organ/cyberimp/arm/lighter
 	category = list("Misc", "Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/lethalsyringe
```
Comment (nednaZ) [inline monkestation/code/modules/research/designs/medical_designs.dm:23]: If these are going to be usable for more than lethal injection now, they could probably use a rename.
One idea is "Titanium Syringe", just going by the materials used.
I've seen one server call them iron spikes.

**Review by nednaZ — CHANGES_REQUESTED** [review]
Looks fine outside the two changes.

File: code/modules/projectiles/guns/misc/syringe_gun.dm (hunk 3 above)
Comment (KoboldCommando) [reply code/modules/projectiles/guns/misc/syringe_gun.dm:64]: You're right.  When you're staring at a file and immersed in variables like A there, written without being expected to be used more than one line lower, you kinda lose sight easily lol.

**Comment by KoboldCommando** [comment]
Lethal Injection Syringe is changed to Metamaterial Syringe, reflecting its relation to the metamaterial beaker tech-wise.  There's also a nod toward the larger injection dosage.  I changed the path as well which touches just a few maps.

**Review by nednaZ — APPROVED** [review]
Looks good to me

**Pull request merged**